frontend/src/components/admin/userManagement/UserManagement.js

```diff
@@ -1,8 +1,13 @@
-import React, { useContext, useState, useEffect, useRef } from "react";
+import React, {
+  useContext,
```

Comment:
i really dont think you needed to make any UI change unless you were adressing a different issue other than improving pagination response